analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/DisablingRequestValidationBase.cs

```diff
@@ -74,25 +74,25 @@ private void CheckController(SymbolAnalysisContext c)
             }
         }
 
-        private void CheckWebConfig(SonarAnalysisContext context, CompilationAnalysisContext c)
+        private void CheckWebConfig(SonarCompilationAnalysisContext c)
```

Comment:
It would be better to have this named `context`.  
```suggestion
        private void CheckWebConfig(SonarCompilationAnalysisContext context)
```